mozjs/python/mozbuild/mozbuild/test/frontend/data/rust-library-name-mismatch/Cargo.toml

```diff
@@ -1,12 +0,0 @@
-[package]
```

Comment:
Cargo will complain about malformed toml files and refuse to look any further.

I upstreamed this into the `js/src/make-source-package.sh` bundling script that creates the source release tarballs. If we ever want to point directly at a revision of m-c, rather than snapshots in this repo, then we will need another fix.